src/routes/userRoutes.js

```diff
@@ -35,5 +35,52 @@ routes.resetPassword = async (req, res) => {
   }
 }
 
+routes.createUser = async (req, res) => {
+  const userInfo = {
+      username: req.body.username,
+      email: req.body.email,
+      password: req.body.password
+  }
+  try {
+      await signUp(userInfo)
+      logger.info('Succeded creating user account', userInfo.username)
+      return res.status(200).json('Succeded creating user account')
+  } catch (err) {
+      logger.error("Creating user failed", userInfo.username, err)
+      return res.status(500).json({error: {message: 'Creating user failed. Please try again'}})
+  }
+}
+
+routes.deleteUser = async (req, res) => {
+  if (!req.params.id) {
+      logger.info("user id was not provided")
+      return res.status(400).json({error: {message: "user id was not provided"}})
+  }
+  const {Accounts} = db.getModels()
+  try {
+      const account = await Accounts.findOne({
+      where: {
+          id: req.params.id
+      }
+      })
+      if (!account) {
+      logger.info("Cannot find user", req.params.id)
+      res.status(404).json({error: {message: "Cannot find user"}})
+      return
+      }
+      if (account.username !== req.session.username) {
+          logger.error("Username does not match to cookie", account.username, req.session.username)
+          res.status(500).json({error: {message: "Username does not match to cookie"}})
```

Comment:
could you return a 403 this action should be forbidden by a user 
```
res.status(403)
```
500 means there is something wrong with the server https://en.wikipedia.org/wiki/List_of_HTTP_status_codes